external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,560 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
+
+The format of the `cell_uri_template` value **must** follow [RFC6570](https://www.rfc-editor.org/rfc/rfc6570) guidance for URI Templates. In the case of any doubt, follow the pattern in the examples shown above (i.e. `http://example.org/some-uri/{+column_name}`), as this will ensure csvcubed safely [transforms the column header](../uris.md#csv-column-name-safe-transformation) to the CSV-W format.
 
 ## Attributes Configuration
 
+The following fields can be configured for each attribute in your data set.
+
 | **field name**           | **description**                                                                                                                                                                                                                                                                                                                                                                | **default value**                                                                |
 |--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
 | `type`                   | The type of the column; to configure an attribute column use the value `attribute` (Required)                                                                                                                                                                                                                                                                                  | *dimension*                                                                      |
 | `from_template`          | Use a [column template](templates.md)                                                                                                                                                                                                                                                                                                                                          | *none*                                                                           |
 | `label`                  | The title of the column (Optional)                                                                                                                                                                                                                                                                                                                                             | The capital case of the header in the csv file with spaces replacing underscores |
```

Comment:
I'm not sure this is techically true at the moment. See https://github.com/GSS-Cogs/csvcubed/issues/759